index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+
+      <div class="title-wrap">
+        <h1 id="title">Nikola Tesla</h1>
+      </div>
+
+      <div id="img-div">
+        <img id="image" src="images/nikola-tesla.jfif" alt="Nikola Tesla">
+        <p id="img-caption">Image of Nikola Tesla</p>
+      </div>
```

Comment:
I think it would be better to use the HTML5 semantic tag _header_ to group the title and image.

HTML5 semantic tags will help to more accurately describe the relationship between HTML elements in your code.

Here, the header element can be used to group introductory content such as the title and the image.

The header element should also be placed before the main element.